instrumentation/jetty-httpclient/jetty-httpclient-9.2/library/src/main/java/io/opentelemetry/instrumentation/jetty/httpclient/v9_2/JettyClientTracingBuilder.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.instrumentation.jetty.httpclient.v9_2;
+
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.instrumentation.api.instrumenter.Instrumenter;
+import io.opentelemetry.instrumentation.jetty.httpclient.v9_2.internal.JettyClientInstrumenterBuilder;
+import org.eclipse.jetty.client.HttpClientTransport;
+import org.eclipse.jetty.client.api.Request;
+import org.eclipse.jetty.client.api.Response;
+import org.eclipse.jetty.util.ssl.SslContextFactory;
+
+public final class JettyClientTracingBuilder {
+
+  private final OpenTelemetry openTelemetry;
+  private HttpClientTransport httpClientTransport;
+  private SslContextFactory sslContextFactory;
+
+  public JettyClientTracingBuilder(OpenTelemetry openTelemetry) {
+    this.openTelemetry = openTelemetry;
+  }
+
+  public JettyClientTracingBuilder setHttpClientTransport(HttpClientTransport httpClientTransport) {
```

Comment:
We can fill in other javadoc later but do you think you can add to at least these two Jetty-specific ones being closest to the code?

Reply:
Sure - I can put in some good javadoc for these